src/cascadia/TerminalConnection/TerminalConnection.vcxproj

```diff
@@ -8,6 +8,9 @@
     <SubSystem>Console</SubSystem>
     <OpenConsoleUniversalApp>true</OpenConsoleUniversalApp>
   </PropertyGroup>
+  <PropertyGroup Label="Vcpkg">
+    <VcpkgEnabled>false</VcpkgEnabled>
+  </PropertyGroup>

```

Comment:
This change seems unrelated? Can we revert it?

Reply:
ok